layout/layout_test.go

```diff
@@ -32,13 +32,26 @@ func TestLayout(t *testing.T) {
 	assert.Equal(t, &tele.LongPoller{}, pref.Poller)
 	assert.Equal(t, pref, ltfs.Settings())
 
-	assert.ElementsMatch(t, []tele.Command{{
+	assert.Equal(t, []tele.Command{{
 		Text:        "start",
 		Description: "Start the bot",
 	}, {
 		Text:        "help",
 		Description: "How to use the bot",
+	}, {
+		Text:        "settings",
+		Description: "{{ text `cmd_settings` }}",
```

Comment:
previous pr had a question about testing this part of template
https://github.com/tucnak/telebot/pull/681#discussion_r1711361203

The reasons to keep `lt.Commands` in tests:
1) make it clear what happens when you use templates with `lt.Commands()`
2) the test already contained testing of both Commands and CommandsLocale functions, kept it here

Reply:
@demget should I check anything else before you run CI workflow?